src/rpc/net.cpp

```diff
@@ -290,35 +290,33 @@ static RPCHelpMan addnode()
                 },
         [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
 {
-    std::string strCommand;
-    if (!request.params[1].isNull())
-        strCommand = request.params[1].get_str();
-    if (strCommand != "onetry" && strCommand != "add" && strCommand != "remove") {
+    const std::string command{request.params[1].get_str()};
```

Comment:
The title and commit messages don't fit: This PR changes concerns `addnode`, not `getpeerinfo`.

Reply:
Ooops, I think things changed during the way, thanks for notice it! Done!